frontend/src/lib/components/AppEditorLink/UrlRow.js

```diff
@@ -72,7 +72,7 @@ export function UrlRow({ actionId, url, saveUrl, deleteUrl, allowNavigation }) {
                         data-attr="app-url-item"
                         href={appEditorUrl(actionId, editedValue)}
                         target="_blank"
-                        rel="noreferrer noopener"
+                        rel="noopener"
```

Comment:
This leads to this ESLint error:

```
##[error]  74:25  error  Using target="_blank" without rel="noreferrer" is a security risk: see https://html.spec.whatwg.org/multipage/links.html#link-type-noopener  react/jsx-no-target-blank
```

Reply:
Yep, will disable the noreferrer rule, since we actually usually _should_ reveal that visits are coming from PostHog (as it's basically internal traffic and should be possible to filter out from events).

Reply:
Unfortunately ESLint didn't want to only enforce `noopener`, but the security risk if someone forgets to use it _should_ be negligible in our case (as we don't really have links to page other than than posthog.com or the logged-in user's ones).